electronicparsers/fhiaims/parser.py

```diff
@@ -2063,6 +2064,11 @@ def parse_method(self):
         sec_method.electronic = sec_electronic
         sec_electronic.method = 'DFT'
 
+        # Scf threshold energy change
+        sec_scf = Scf()
+        sec_method.scf = sec_scf
+        sec_scf.threshoold_energy_change = self.out_parser.get('scf_threshold_energy_change')
```

Comment:
This doesn't look correct, I think we need different key, just check what key is `sc_accuracy_etot` value saved into and use that, also we need to set proper units (but right now I'm unsure about the units).

Reply:
threshoold_energy_change  typo